components/builder-depot/src/server.rs

```diff
@@ -865,7 +869,7 @@ fn upload_package(req: &mut Request) -> IronResult<Response> {
         }
     }
 
-    let filename = depot.archive_path(&ident, &target_from_artifact);
```

Comment:
Ohh, is this why the change was made above to `archive_name`?  Because you specifically don't want it to return an absolute path any more?  Because you're getting your absolute path here, from `file_path`?